### PR TITLE
Add initialize method

### PR DESCRIPTION
**Pull request by nikostoulas**
initialize-script

### CODE REVIEW

File: src/default-options.ts [hunk 1]
```diff
@@ -15,5 +15,6 @@ export default {
   },
   beforeMiddleware: [],
   afterMiddleware: [],
-  beforeStart: []
+  beforeStart: [],
+  initialize: null
```
Comment (nikostoulas) [inline src/default-options.ts]: @anastasios-workable This overlaps with beforeStart. Initialize runs after diamorphosis while beforeStart is another queued function that is run after middlewares are initialized. Wdyt?

Comment (nikostoulas) [reply src/default-options.ts]: TLDR I ported orka into cerberus and beforeStart was not enough

Comment (anastasios-workable) [reply src/default-options.ts]: I really think that diamorphosis should be moved out, it complicates things a lot and is not that much boilerplate tbh. Thoughts?

Comment (nikostoulas) [reply src/default-options.ts]: I would rather not.  However, apps that would need initialize can call diamorphosis outside orka. Tested this in cerberus so I will drop this pr.

**Pull request closed without merging**